### PR TITLE
feat(AN-1046): update user buffering calculation

### PR DESCRIPTION
**Pull request by diamond-darrell**
https://kaltura.atlassian.net/browse/AN-1046

### CODE REVIEW

File: src/app/modules/entry-live/views/live-discovery-table/devices-table/live-discovery-devices-table.config.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export class LiveDiscoveryDevicesTableConfig extends ReportDataBaseConfig {
             sortOrder: 2,
           },
           'view_unique_buffering_users': {
```
Comment (amirch1) [inline src/app/modules/entry-live/views/live-discovery-table/devices-table/live-discovery-devices-table.config.ts]: @diamond-darrell - metric name should probably be view_buffer_time_ratio

**Comment by amirch1** [comment]
@diamond-darrell - Summary row gets empty value

File: src/app/modules/entry-live/views/live-discovery-table/devices-table/live-discovery-devices-table.config.ts (hunk 1 above)
Comment (diamond-darrell) [reply src/app/modules/entry-live/views/live-discovery-table/devices-table/live-discovery-devices-table.config.ts]: fixed

**Comment by amirch1** [comment]
@diamond-darrell - lets round up the numbers:
image

**Comment by diamond-darrell** [comment]
> @diamond-darrell - lets round up the numbers:
> image

fixed

**Pull request merged**